jep/302/README.adoc

```diff
@@ -163,32 +210,58 @@ In the context of _Essentials_, we cannot rely on external backups to revert to
 
 == Reasoning
 
+=== Scope of the data snapshotting: not a backup system
+
+The practical timeframe where this system is designed to be used is in the next seconds or minutes after an upgrade occurred.
+If Jenkins, after it has been restarted, is deemed unhealthy, then an auto-rollback _can_ be initiated.
+
+If a version is proved to be problematic after a few days, the data snapshotting system will **not** be used.
+
+This would be quite impractical because the instance probably generated actual work items during this timeframe.
+So rolling back that much later would be subject to losing data.
```

Comment:
grammar: _would risk data loss_

Reply:
also: _rolling that back_ (separable verbs, sigh)

Reply:
done thanks